api/envoy/extensions/access_loggers/wasm/v3/wasm.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package envoy.extensions.access_loggers.wasm.v3;
```

Comment:
If you plan on doing churn here this should be either v3alpha or have the `option (udpa.annotations.file_status).work_in_progress = true;` annotation.

Reply:
I don't have any plans of churning.  If we need to make any changes we can do so in v4.